packages/gestalt-design-tokens/tokens/space/base.json

```diff
@@ -3,17 +3,27 @@
     "0": { "value": "0" },
     "100": { "value": "4px" },
     "200": { "value": "8px" },
-    "300": { "value": "16px" },
-    "400": { "value": "24px" },
-    "500": { "value": "32px" },
-    "600": { "value": "64px" },
+    "300": { "value": "12px" },
+    "400": { "value": "16px" },
+    "500": { "value": "20px" },
+    "600": { "value": "24px" },
+    "700": { "value": "28px" },
+    "800": { "value": "32px" },
+    "900": { "value": "36px" },
+    "1000": { "value": "40px" },
+    "1100": { "value": "44px" },
+    "1200": { "value": "48px" },
+    "1600": { "value": "64px" },
     "negative": {
       "100": { "value": "-4px" },
       "200": { "value": "-8px" },
-      "300": { "value": "-16px" },
-      "400": { "value": "-24px" },
-      "500": { "value": "-32px" },
-      "600": { "value": "-64px" }
+      "300": { "value": "-12px" },
+      "400": { "value": "-16px" },
+      "500": { "value": "-20px" },
+      "600": { "value": "-24px" },
+      "700": { "value": "-28px" },
+      "800": { "value": "-32px" },
```

Comment:
why are we skipping negative from 900 to 1600? Shouldn't all mirror? Can we document / comment in the code if there's a reason why?

Reply:
Good catch, added them back for consistency after chatting with Hector. Thinking was because there was no usage of the negative tokens, but we can revisit that later in the docs and keep the tokens